packages/lodestar/src/eth1/impl/ethers.ts

```diff
@@ -108,9 +106,11 @@ export class EthersEth1Notifier extends (EventEmitter as { new(): Eth1EventEmitt
     }
     this.provider.removeAllListeners("block");
     this.started = false;
+    this.logger.verbose("Eth1 notifier is stopping");
     while (this.processingBlock) {
       await sleep(5);
     }
```

Comment:
I don't think we need this. Even if we stop eth1, processingBlock will still finish eventually because node won't kill process until something is still executing.